sqlstore/src/main/resources/db/migration/V8200.sql

```diff
@@ -0,0 +1,28 @@
+-- serviceboards
+
+-- Create table for instrument ServiceRecords
+CREATE TABLE Instrument_ServiceRecord (
+  recordId bigint(20) NOT NULL AUTO_INCREMENT,
+  instrumentId bigint(20) NOT NULL,
+  PRIMARY KEY (recordId),
+  KEY sequencerServiceRecord_sequencer_fkey (instrumentId)
```

Comment:
Slightly different syntax for the foreign key constraint declaration at table creation time:
`CONSTRAINT fk_instrumentServiceRecord_instrument FOREIGN KEY (instrumentId) REFERENCES Instrument (instrumentId)`

(It looks like you're adding the foreign key on line 21 as well. Either adding it at table creation time or after is valid, but probably shouldn't be both.)